@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -130,17 +139,13 @@ export const ProgressBar = forwardRef<HTMLDivElement, ProgressBarProps>(
       >
         <div
           className={cl("navds-progress-bar__foreground", {
-            "navds-progress-bar__foreground--indeterminate": Number.isInteger(
-              simulated?.seconds,
-            ),
+            "navds-progress-bar__foreground--indeterminate": simulated?.seconds,
```

Comment:
@larseirikhansen Will this break anything? Should 0 seconds be a valid entry here?